Assets/Scripts/Card.cs

```diff
@@ -21,5 +37,21 @@ public void OnDrag(PointerEventData eventData)
     public void OnEndDrag(PointerEventData eventData)
     {
         transform.SetParent(parentAfterDragging);
+        var results = new System.Collections.Generic.List<RaycastResult>();
+        EventSystem.current.RaycastAll(eventData, results);
+        foreach (var ray in results)
+        {
+            if (ray.gameObject != gameObject && ray.gameObject.CompareTag("Card"))
+            {
+                ray.gameObject.GetComponent<Card>().OnCardMerge();
```

Comment:
Something we may want to tweak later (depending on how playtesting goes) is whether cards need to be grabbed and "stopped" (during falling) to be able to actually combine them. But this might work just as well